src/ERC1967Proxy.ts

```diff
@@ -3,12 +3,15 @@ import { bytesToString, toBytes } from "viem";
 
 ponder.on("ERC1967Proxy:Transfer", async ({ event, context }) => {
   const { Fname } = context.entities;
+
+  const fid = await context.contracts.IdRegistry.read.idOf([event.params.to]);
```

Comment:
the association between the fid and fname is "broken" if the fid is transferred out to a different address. i don't think this handles that case since it permanently associated the fname with the fid.

Reply:
ah good point. just to make sure I follow - are you saying that in the `IdRegistry:Transfer` handler we'd have to update the `fid` of all names owned by the `event.params.from` address?

Reply:
yep, that's right